models/nxos/ospf_global/nxos_ospf_global.yaml

```diff
@@ -0,0 +1,529 @@
+                          type: bool
+                        default_information_originate:
+                          description:
+                            - Originate Type-7 default LSA into NSSA area.
+                          type: bool
+                        no_redistribution:
+                          description:
+                            - Do not send redistributed LSAs into NSSA area.
+                          type: bool
+                        no_summary:
+                          description:
+                            - Do not send summary LSAs into NSSA area.
+                          type: bool
+                        translate:
+                          description:
+                            - Translate LSA.
+                          type: dict
+                          suboptions:
+                            type7:
+                              description:
+                                - Translate from Type 7 to Type 5.
+                              type: dict
+                              suboptions:
+                                always:
+                                  description:
+                                    - Always translate LSAs
+                                  type: bool
+                                never:
+                                  description:
+                                    - Never translate LSAs
+                                  type: bool
+                                supress_fa:
+                                  description:
+                                    - Suppress forwarding address in translated LSAs.
+                                  type: bool
+                    range:
+                      description:
+                        - Configure an address range for the area.
+                      type: dict
+                      suboptions:
+                        prefix:
+                          description:
+                            - IP in Prefix format (x.x.x.x/len)
+                          type: str
+                        cost:
+                          description:
+                            - Cost to use for the range.
+                          type: int
+                        not_advertise:
+                          description:
+                            - Suppress advertising the specified range.
+                          type: bool
+                    stub:
+                      description:
+                        - Settings for configuring the area as a stub.
+                      type: dict
+                      suboptions:
+                        enable:
+                          description:
+                            - Configure the area as a stub.
+                          type: bool
+                        no_summary:
+                          description:
+                            - Prevent ABR from sending summary LSAs into stub area.
```

Comment:
Can you mention the exact behavior when the value is set as 'False' and 'True'. ?

Reply:
+1